plugins/cilium-cni/cni-install.sh

```diff
@@ -174,10 +174,41 @@ EOF
 	;;
 
 "aws-cni")
-	jq --argjson 'ENABLE_DEBUG' "$ENABLE_DEBUG" --arg 'LOG_FILE' "$LOG_FILE" \
-  	  '.plugins += [{"name": "cilium", "type": "cilium-cni", "enable-debug": $ENABLE_DEBUG, "log-file": $LOG_FILE}]' \
-  	  "${CNI_CONF_DIR}/10-aws.conflist.cilium_bak" > "${CNI_CONF_NAME}"
-	;;
+  if [ -f "${CNI_CONF_DIR}/10-aws.conflist.cilium_bak" ]; then
+    jq --argjson 'ENABLE_DEBUG' "$ENABLE_DEBUG" --arg 'LOG_FILE' "$LOG_FILE" \
+          '.plugins += [{"name": "cilium", "type": "cilium-cni", "enable-debug": $ENABLE_DEBUG, "log-file": $LOG_FILE}]' \
+          "${CNI_CONF_DIR}/10-aws.conflist.cilium_bak" > "${CNI_CONF_NAME}"
+      ;;
+  else
+    cat > "${CNI_CONF_NAME}" <<EOF
+{
+  "cniVersion": "0.3.1",
+  "name": "aws-cni",
+  "plugins": [
+    {
+      "name": "aws-cni",
+      "type": "aws-cni",
+      "vethPrefix": "eni",
+      "mtu": "9001",
+      "pluginLogFile": "/var/log/aws-routed-eni/plugin.log",
+      "pluginLogLevel": "DEBUG"
+    },
+    {
+      "type": "portmap",
+      "capabilities": {"portMappings": true},
+      "snat": true
+    },
+    {
+       "name": "cilium",
+       "type": "cilium-cni",
+       "enable-debug": ${ENABLE_DEBUG},
+       "log-file": "${LOG_FILE}"
+    }
+  ]
+}
+EOF
+	  ;;
+  fi
```

Comment:
I think commit `cni: Provide default for aws-cni chaining config` should be merged with the previous commit to avoid the number of touched code lines (a pain when git blaming).

Reply:
Same for commit `cni: Fix install script format`.